api/src/test/java/com/epam/pipeline/manager/pipeline/PipelineRunManagerTest.java

```diff
@@ -91,12 +86,11 @@
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-@DirtiesContext //TODO: find a better workaround, this may make tests slower. Maybe, create a special package for
-                // integration tests, so they will be executed one after another and the context will remain?
+// TODO: find a better workaround, this may make tests slower. Maybe, create a special package for
+// integration tests, so they will be executed one after another and the context will remain?
```

Comment:
Let's remove the comment along with the annotation because we've resolved the corresponding todo.

Reply:
Done!